packages/node-cli/messages.json

```diff
@@ -45,18 +45,18 @@
 	"COMMAND_CREATEPROJECT_OVERWRITE_PROJECT": "Do you want to overwrite the {0} project? All the files and folders from the former project will be deleted and replaced by the new project.",
 	"COMMAND_CREATEPROJECT_WRONG_PROJECT_TYPE": "Invalid \"type\" specified. The expected values are the following - \"ACCOUNTCUSTOMIZATION, SUITEAPP\".",
 
-	"COMMAND_DEPLOY_ERRORS_APPLY_CONTENT_PROTECTION_IN_ACP": "Content protection can not be applied in an Account Customziaton Project.",
+	"COMMAND_DEPLOY_ERRORS_APPLY_INSTALLATION_PREFERENCES_IN_ACP": "Installation preferences can not be applied in an Account Customizaton Project.",
 	"COMMAND_DEPLOY_ERRORS_WRONG_ACCOUNT_SPECIFIC_VALUES_OPTION": "You have specified an invalid value for the --accountspecficvalue option. Enter either WARNING or ERROR.",
 	"COMMAND_DEPLOY_QUESTIONS_ACCOUNT_SPECIFIC_VALUES": "What do you want to do if the project contains account-specific values?",
-	"COMMAND_DEPLOY_QUESTIONS_APPLY_CONTENT_PROTECTION": "Do you want to apply content protection?",
+	"COMMAND_DEPLOY_QUESTIONS_APPLY_INSTALLATION_PREFERENCES": "Do you want to apply installation preferences?",
 	"COMMAND_DEPLOY_QUESTIONS_CHOICES_ACCOUNT_SPECIFIC_VALUES_CANCEL_PROCESS": "Cancel the process",
 	"COMMAND_DEPLOY_QUESTIONS_CHOICES_ACCOUNT_SPECIFIC_VALUES_DISPLAY_WARNING": "Display a warning",
 	"COMMAND_DEPLOY_QUESTIONS_PERFORM_LOCAL_VALIDATION": "Do you want to validate locally before deploying?",
-	"COMMAND_DEPLOY_MESSAGES_APPLYING_CONTENT_PROTECTION": "The \"{0}\" project has been deployed applying content protection.",
+	"COMMAND_DEPLOY_MESSAGES_APPLYING_INSTALLATION_PREFERENCES": "The \"{0}\" project has been deployed applying installation preferences.",
 	"COMMAND_DEPLOY_MESSAGES_DEPLOYING": "Deploying project \"{0}\" with authID \"{1}\"...",
 	"COMMAND_DEPLOY_MESSAGES_LOCALLY_VALIDATED": "The \"{0}\" project has been locally validated.",
-	"COMMAND_DEPLOY_MESSAGES_NOT_APPLYING_CONTENT_PROTECTION": "The \"{0}\" project has been deployed without applying content protection.",
-	"COMMAND_DEPLOY_MESSAGES_NOT_ASKING_CONTENT_PROTECTION_REASON": "Content protection was not applied because the hiding.xml and locking.xml files were not found in the \"InstallationPreferences\" folder of the project. You need to create both files in order to apply content protection.\nTo learn how to create a hiding.xml file, see {0}.\nTo learn how to create a locking.xml file, see {1}.",
+	"COMMAND_DEPLOY_MESSAGES_NOT_APPLYING_INSTALLATION_PREFERENCES": "The \"{0}\" project has been deployed without applying installation preferences.",
+	"COMMAND_DEPLOY_MESSAGES_NOT_ASKING_INSTALLATION_PREFERENCES_REASON": "Installation preferences were not applied because the hiding.xml, locking.xml and overwriting.xml files were not found in the \"InstallationPreferences\" folder of the project. You need to create the three files in order to apply installation preferences.\nFor more information about installation preferences, see {0}",
```

Comment:
As far as I know, the ovewritting file is not a mandatory one. We did in this way to not break the deployment of the old projects without that file.

Reply:
overwriting is definitively not mandatory.